### ISSUE TITLE
Enable multiple stacking instances

### ISSUE THREAD

**Issue by jasperjansz** [blockstack/stacks-wallet#631]
I think people should be able to stack multiple times simultaneously if they have the funds. Of the top of my head, this would mean the start stacking CTA on the homepage (and possibly menu) would be persistent and we'd need some overview of stacking.

**Comment by GinaAbrams** [comment]
@jasperjansz would this also cover allowing for multiple stacking instances across the same wallet?

**Comment by aulneau** [comment]
On a technical level I don't believe this is currently possible.

**Comment by kyranjamie** [comment]
Indeed, I'm pretty sure the contract only allows stacking once per address. We should think about supporting switching between multiple wallets though.

**Comment by fluidvoice** [comment]
> @jasperjansz would this also cover allowing for multiple stacking instances across the same wallet address? As in they would be able to add balances via the same wallet address for future cycles?

@GinaAbrams I'm confused, didn't you recently post instructions for a workaround for doing exactly this?... adding STX to a wallet currently already locked-up stacking.  So is that workaround invalid?

**Comment by markmhx** [comment]
@fluidvoice I believe those instructions remain valid. However, they're for adding STX to an entirely new wallet for initiating a second Stacking commitment in parallel with the first commitment on the initial wallet.
